Mathlib/Algebra/MvPolynomial/Basic.lean

```diff
@@ -1539,7 +1539,6 @@ theorem aeval_X (s : σ) : aeval f (X s : MvPolynomial _ R) = f s :=
   eval₂_X _ _ _
 #align mv_polynomial.aeval_X MvPolynomial.aeval_X
 
-@[simp]
 theorem aeval_C (r : R) : aeval f (C r) = algebraMap R S₁ r :=
```

Comment:
What is the reasoning behind this? These look fine to me at first glance.

Reply:
With the new `algHom_C`, this is provable by `simp`.